helper/start-bash-session.sh

```diff
@@ -0,0 +1,8 @@
+# Start an interactive bash session in the current bash environment.
```

Comment:
Looks like this is currently unused. What is it for?

Reply:
It's used in the mempool flake:
https://github.com/erikarvstedt/nix-bitcoin-mempool/blob/a47cc2d5a1c08b32eee79fa079bbadbc7bf5cf57/pkgs/mempool/generate.sh#L6

Reply:
Could you add a comment explaining how this is related to nix-bitcoin if it's unused in nix-bitcoin ("useful for flakes" or something)?

Reply:
Done.